src/AppInstallerCLICore/Workflows/InstallFlow.h

```diff
@@ -28,6 +28,12 @@ namespace AppInstaller::CLI::Workflow
     // Inputs: InstallationNotes
     // Outputs: None
     void DisplayInstallationNotes(Execution::Context& context);
+    
+    // Checks if there are any included arguments that are not supported for the package.
+    // Required Args: None
+    // Inputs: Installer, UnsupportedArguments

```

Comment:
I always took these comments as talking about what data is in the context. Like "this function will call `context.get<Input>()` and after it's done you can do `context.get<Output>()`". So I'd only list `Installer` and not `UnsupportedArguments` because there isn't an entry like that.

```suggestion
    // Inputs: Installer
```

Reply:
Changed to only listing `Installer`